staging/src/k8s.io/legacy-cloud-providers/azure/azure_backoff.go

```diff
@@ -197,6 +197,12 @@ func (az *Cloud) CreateOrUpdateSecurityGroup(service *v1.Service, sg network.Sec
 		if resp != nil && resp.StatusCode == http.StatusPreconditionFailed {
 			az.nsgCache.Delete(*sg.Name)
 		}
+
+		// Invalidate the cache because another new operation has canceled the current request.
+		if strings.Contains(err.Error(), "CanceledAndSupersededDueToAnotherOperation") {
```

Comment:
add a const string for this since other places also uses this string, and what's the full error msg? could you paste here?

Reply:
The full error mssage is 

```
Code="Canceled" Message="Operation was canceled." Details=[{"code":"CanceledAndSupersededDueToAnotherOperation","message":"Operation PutLoadBalancerOperation (c29592fc-da61-47c3-b032-b4a2df0cc62a) was canceled and superseded by operation PutLoadBalancerOperation (b0a47269-40c5-461a-b704-2f162f8300f3)."}]
```

Reply:
Good catch. Fixed in the new commit.